src/v2/Apps/Order/Components/SavedAddressItem.tsx

```diff
@@ -88,3 +95,8 @@ const EditButton = styled(Text)`
     text-decoration: underline;
   }
 `
+
+const StyledPill = styled(Pill)`
+  background-color: ${color("black10")} !important;
+  pointer-events: none;
+`
```

Comment:
Blocking: As this is a component in our design system the more constrained the better, meaning we should avoid styling palette components in this way. If we need to add the ability to control background colors we should update the palette `Pill` component directly. But more ideally, we push back at design on these one-off changes and instead go with current defaults.

Reply:
Yes, totally agree with this.  What should I do at the moment? Leave it to work like this? As I understand I can't make corrections for the Pill component at Palette by myself.

https://user-images.githubusercontent.com/55637696/125597431-cb2abeb5-86e8-4d2d-aefd-956c5bf2583d.mov

Reply:
I would suggest shipping this for now, and then we can iterate on adding a new property to Palette! Thoughts @pepopowitz?  

It would be worth raising in the #design-system channel in our slack so that designers (and other devs) can comment.

Reply:
A few thoughts: 

- +1 to shipping & iterating.
- Is there a bug here somewhere? In the video (1) the Pill starts with a medium gray background; (2) it gets a very light gray background when we hover over the second address box; (3) it stays light gray when you hover over the Pill directly; (4) it changes to a medium gray background when leaving the Pill but still hovering the address box. I'd expect 4 & 2 to match states, but they don't. If there's a bug within this PR that's causing it to turn light gray in step 2, I don't think the video would look as bad. (Is a hover state being propagated to the Pill somehow?) On the other hand, if the bug is within palette, we should track it there but I wouldn't expect to hold up this PR for it.
- It seems especially important to call out this exact demo to the design system group. The reason the default `Pill` styling doesn't look great is the combination of the Pill with an area that's turning gray due to hover. I'd expect the group to come back with either (1) we need an alternate style for the Pill because we'll be doing this more, or (2) this current UI could be re-worked to better utilize the components & design we already have, avoiding the need for an alternate `Pill` style. 
- It also would be worth raising to @tinn, to collaboratively work toward a short-term and long-term solution that utilizes palette completely instead of dropping down into styled-components to override the background. 

That's a lot, so to be clear, I wouldn't personally hold up a merge of either the styled-components solution or the solution in this video, as long as we track the work to follow up and make it better.

Reply:
For reference, here's a link to [Palette's Pill storybook](https://palette-storybook.artsy.net/?path=/story/components-pill--states)

> as long as we track the work to follow up and make it better

I'd still encourage us to avoid merging the overrides with `!important` for the time being, unless there's a ticket and an immediate follow-up. The visual hover state is weird but it's noticeable, where stray overrides and `!important` could cause real confusion down the road, as they tend to do.

Reply:
I'm going to take a stab at writing up a ticket for this and pull it into the sprint as a follow up! Will ping you @pepopowitz to make sure it's accurate. 🙏